src/status_im/navigation/core.cljs

```diff
@@ -92,6 +92,12 @@
  (fn [] (gesture/gesture-handler-root-hoc views/bottom-sheet))
  (fn [] views/bottom-sheet))
 
+;;;; Alert Banner
+(navigation/register-component
+ "alert-banner"
+ (fn [] (gesture/gesture-handler-root-hoc views/alert-banner))
+ (fn [] views/alert-banner))
+
 ;; LEGACY (should be removed in status 2.0)
```

Comment:
can you move this legacy comment further down, it's unclear if it's for above or below